cpp/ps0001/the second task.cpp

```diff
@@ -1,26 +1,24 @@
-
-
-
 #include <iostream>
 
 int main()
 
 {
-    int firstnumber , secondnuber , thirdnumber ;
+    // prompting users for 3 numbers to add 1/2
+    short firstnumber , secondnuber , thirdnumber ;
     
-    std::cout << "inter the first number\n";
+    std::cout << "enter the first number\n";
     std::cin >> firstnumber;
 
-    std::cout << "inter the second number\n";
+    std::cout << "enter the second number\n";
     std::cin >> secondnuber;
 
-    std::cout << "inter the thirdnumber number\n";
+    std::cout << "enter the thirdnumber number\n";
     std::cin >> thirdnumber;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add